clang/test/Driver/sycl.c

```diff
@@ -1,3 +1,6 @@
+// Failing on Windows - temporarily disable
+// REQUIRES: system-linux
```

Comment:
I don't suggest making any changes to this patch, but I have a general recommendation.
The better way to disable the test on one platform is to mark is as unsupported on that platform (e.g. `UNSUPPORTED: system-windows`). 
As of now DPC++ is validated on three different platforms Windows, Linux and MacOS. MacOS validation doesn't cover clang lit tests, so existing approach provides the desired result, but potentially reduces the testing scope on supported platforms.